.vscode/settings.json

```diff
@@ -1,3 +1,8 @@
 {
-  "rust-analyzer.cargo.features": ["abort", "stargate", "staking", "cosmwasm_1_3"]
-}
+  "rust-analyzer.cargo.features": [
+    "abort",
+    "stargate",
+    "staking",
+    "cosmwasm_1_4"
+  ]
+}
```

Comment:
Do you have an [editorconfig](https://editorconfig.org/) plugin installed? This should avoid those missing line breaks amonst other things

Reply:
Now, I do :P